### ISSUE TITLE
Enhancement: import/export offsets

### ISSUE THREAD

**Issue by jbostoen** [jbostoen/ZTStudio#30]
Offsets could also be imported/exported.
Combine this with the "crop to relevant area" and it's most likely the best performing method if people don't do anything with the images except for recoloring.

**Comment by HENDRIX-ZT2** [comment]
yeah, offset could essentially be written into the file name like:
E0001_X-12_Y23.png

But I don't think many designers actually recolor like that any more. It seems to be either palette or new sprites altogether.

**Comment by jbostoen** [comment]
Your script really opened things up after I got this far together with Vondell's Blender skills, HENDRIX-ZT2!

I added this idea here because I recently talked to a new designer who still keeps track of offsets (non-recolor related though). Would be nice if it's done automatically. And on a side note: I'd still hope to found out the #11 Mystery Bytes. That info could also be written down. Perhaps even together with height/width (although you could easily derive this from the PNGs as well).

The idea was to write this info to an INI file instead.

**Comment by jbostoen** [comment]
It's likely going to work as an extra option (kind of wondering whether I should implement it by default for some PNG export methods).

Per graphic/view (SE, SW, ...); there will be an SE.nfo, SW.nfo file; containing some details about the frame (offsets, height/width in ZT1 Graphic, the "mystery bytes" I'd still like to uncover, ...
It will also contain some info on the actual graphic.

It should also allow for better re-import.
I'd like to keep it separated from the filenames of the graphics.
